app/src/main/java/com/neo/speaktouch/utils/NodeValidator.kt

```diff
@@ -97,7 +97,7 @@ object NodeValidator {
 
         if (mustReadContent(node)) return false
 
-        return isClickable(node) && hasReadableChild(node)
+        return hasInteraction(node) && hasReadableChild(node)
```

Comment:
This fixes the bug but causes another one. It is focusing on the RecyclerView of the 'history' section of the Kotatsu app, a behavior that diverges from Talkback. I will do some tests to determine if the validation is wrong or if lists are an exception